src/tile/tile.scss

```diff
@@ -11,6 +11,7 @@
   flex-direction: column;
   width: $iui-3xl * 3;
   box-shadow: $iui-elevation-2;
+  backface-visibility: hidden;
```

Comment:
For me, when this rule is set, text is always a bit blurry:
![PR_comments2](https://user-images.githubusercontent.com/36186912/116034247-a8490000-a66b-11eb-893f-ff6f25af8d58.gif)

Reply:
Same, but it does help on hover...

Reply:
Did you try `will-change: transform` instead?

Reply:
Some other solutions include:
- `transform: translateZ(0)`
- `transform: perspective(1px)`
- `filter: blur(0)`

All from https://stackoverflow.com/questions/14677490/blurry-text-after-using-css-transform-scale-in-chrome

Reply:
will-change does the same, at least at my end..

Reply:
Same for me with other rules - text is always blurry. I think I have tried all of these on tooltip without luck.